src/components/icons/Icons.component.tsx

```diff
@@ -0,0 +1,19 @@
+import facebook from '../../assets/facebook.svg';
+import instagram from '../../assets/instagram.svg';
+import linkedin from '../../assets/linkedin.svg';
+
+import styles from './icons.module.css';
+
+export const IconsComponent = () => (
```

Comment:
3. Implement a dynamic FooterIcon component that includes two props: an icon and a URL. This component should display the passed data as per the design. Use this component to display icons in the footer. Don't forget about typing.